x-pack/legacy/plugins/canvas/public/components/workpad_page/workpad_interactive_page/interaction_boundary.tsx

```diff
@@ -4,6 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import $ from 'jquery';
```

Comment:
I fixed jquery error. But I cannot still understand how it worked properly without that in previous versions.

Reply:
Can you please share what error was this causing?
It seems somewhat unrelated to the other changes in this PR.

Reply:
It's a simple type error caused by jquery $ call below that line.

Somehow, it worked without this in master branch. But it doesn't do so in this branch.